backend/apps/github/index/repository.py

```diff
@@ -32,11 +32,13 @@ class RepositoryIndex(AlgoliaIndex, IndexBase):
         "idx_subscribers_count",
         "idx_top_contributors",
         "idx_topics",
+        "idx_is_archived",
     )
 
     settings = {
         "minProximity": 4,
         "customRanking": [
+            "asc(idx_is_archived)",
```

Comment:
Let's stick to `is_active` for all entities? Typically, if repo is archived is automatically inactive